diesel/src/lib.rs

```diff
@@ -3,6 +3,12 @@
 //! found in the README.
 #![deny(warnings)]
 #![cfg_attr(feature = "unstable", feature(specialization))]
+
+#[macro_use]
+mod macros;
```

Comment:
This got me earlier.  Took me a min to figure out why debug_sql! was not defined inside connection.  👍